buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeMultiplatformExtensionImpl.kt

```diff
@@ -79,6 +79,7 @@ open class AndroidXComposeMultiplatformExtensionImpl @Inject constructor(
                         // We need to set up at least one browser here due to kotlin tooling limitations
                         // Actual browser configuration is set in mpp/karma.config.d/js/config.js
                         useChrome()
+                        useFirefox()
```

Comment:
It enables firefox unconditionally. Is it intentional?

Reply:
@eymar the comment is there though  )
                        // Actual browser configuration is set in mpp/karma.config.d/js/config.js

Reply:
@eymar those are pecularities of how kotling tooling works - this section ignored and we have all browsers set up in our configuration - however the karma helper packages won't be installed otherwise so we need to call this code as well